### ISSUE TITLE
Fix bq2clickhouse factory

### ISSUE THREAD

**Issue by ryscheng** [opensource-observer/oso#2335]
### What is it?

The changes around asset configuration means that the bq2clickhouse jobs aren't running anymore.
Here's an example of some logs
https://admin-dagster.opensource.observer/runs/4115d5f5-aab3-4597-ace8-7c73b8d1b4cb

According to @Jabolol we can just copy the ones from Open Collective here
https://github.com/opensource-observer/oso/blob/28d29b37a4f58f1694d092b1ab87d82829423cd3/warehouse/oso_dagster/assets/open_collective.py#L207
https://github.com/opensource-observer/oso/blob/main/warehouse/oso_dagster/assets/open_collective.py#L642-L644

**Comment by Jabolol** [comment]
Indeed! The fix is to use the appropriate config spec here (following the Open Collective one, for instance). All marts will then materialise as usual.

https://github.com/opensource-observer/oso/blob/28d29b37a4f58f1694d092b1ab87d82829423cd3/warehouse/oso_dagster/assets/clickhouse_dbt_marts.py#L88

**Comment by ryscheng** [comment]
Thanks for fixing this @Jabolol !

**Comment by ryscheng** [comment]
Trying to fix this error:
https://admin-dagster.opensource.observer/runs/b797690a-3e67-4ca4-b2a7-dc396cdc1bee

with this
https://github.com/opensource-observer/oso/pull/2337
